### DISCUSSION TITLE
Feature Request: Displayed Loaded Profile Name

### DISCUSSION THREAD

**Discussion by luxmanifestus** [X2CommunityCore/xcom2-launcher#307] [Ideas]
I think it would be helpful to have the name of the profile currently loaded be displayed, for convenience of certainty.

**Comment by RevZero** [comment]
I don't think this is useful with respect to the way profiles work in AML.
A profile, once loaded, does not stay "active". The moment you add/change mods, you setup will no longer match the profile. And what would you display when you restart AML? The profile that was used last? 
If a profile would stay active and changes would be automatically saved when closing AML, it would probably make sense.
